addons/hr_recruitment_integration_monster/data/hr_contract_status_data.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <function model="hr.contract.status" name="write">
+            <value model="hr.contract.status" search="[
+                ('name', '=', 'Part-Time'),
```

Comment:
Use external identifiers instead of searches based on the name + it won't work if the term is translated.